ui/scripts/system.js

```diff
@@ -31,6 +31,7 @@
 
         if (router.projectid) routerType = _l('label.project');
         if (router.vpcid) routerType = _l('label.vpc');
+        if ("isredundantrouter" in router && router.isredundantrouter) routerType = routerType + " (" + router.redundantstate + ")";
```

Comment:
@rhtyd, Thanks for adding this.
Can we replace these three if statements with if..else? Or do we override the value in this order?

Reply:
@nitin-maharana I'm not sure how could all these three disjoint if conditions be refactored into if..else, what do you have in mind?

Reply:
I'm also not sure. Because in all three statements we set a value to routerType variable according to the condition. If we don't overwrite the variable in the above order, then I think we can put them in if..else condition.

Reply:
@nitin-maharana thanks, here I'm not assuming that redundancy of the router is hard-coded VR being a VPC one only, the if statements are therefore not nested. In future, if we've redundant isolated/shared network VRs I would want the redundant state to be reflected (in parenthesis). I think it's alright to keep as such, unless of course you have a suggestion we can incorporate.